src/commands/main.rs

```diff
@@ -1,19 +1,58 @@
 use serenity::utils::MessageBuilder;
+use serenity::model::OnlineStatus;
+use status::Status;
 
-command!(help(ctx, msg, args) {
+lazy_static! {
+    static ref ALLOWED_USER_IDS: Vec<u64> = vec![76043245804589056, 98633956773093376];
+}
+
+command!(help(ctx, message, args) {
     let _ = ctx;
     let _ = args;
 
-    let message = MessageBuilder::new()
-        .mention(msg.author.clone())
+    let msg = MessageBuilder::new()
+        .mention(message.author.clone())
         .push("\n")
         .push("Commands:\n")
-        .push("`-gen [length]` - ")
-        .push("Generates a message, with an optional length\n")
-        .push("`-genuser \\@mention [length]` - ")
+        .push("`%gen [length] [word]` - ")
+        .push("Generates a message, with an optional length and/or by word\n")
+        .push("`%genuser @mention [length]` - ")
         .push("Generates based on a particular user\n")
-        .push("For more information on Markov chains:")
-        .push("<https://en.wikipedia.org/wiki/Markov_chain>")
+        .push("For more information on Markov chains: ")
+        .push("<https://en.wikipedia.org/wiki/Markov_chain>\n")
+        .push("For more information about the bot: https://github.com/strinking/markov-bot")
         .build();
-    let _ = msg.channel_id.say(&message);
+    let _ = message.channel_id.say(&msg);
+});
+
+command!(status(ctx, message, args) {
+    if ALLOWED_USER_IDS.contains(&message.author.id.0) {
+        ctx.set_presence(None, Status::from_str(args.get(0).unwrap()).unwrap_or(OnlineStatus::Online), false);
+    }
+});
+
+command!(game(ctx, message, args) {
+    if ALLOWED_USER_IDS.contains(&message.author.id.0) {
+        let arg = args.join(" ");
+        ctx.set_game_name(arg.as_str());
+    }
+});
+
+command!(name(ctx, message, args) {
+    if ALLOWED_USER_IDS.contains(&message.author.id.0) {
+        let arg = args.join(" ");
+        let _ = ctx.edit_profile(|p| p.username(arg.as_str()));
+    }
+});
+
+command!(nick(ctx, message, args) {
+    let _ = ctx;
+    if ALLOWED_USER_IDS.contains(&message.author.id.0) {
+        let arg = args.join(" ");
+        if message.is_private() {
+            let _ = ctx.edit_profile(|p| p.username(arg.as_str()));
+        } else {
+            let _ = message.guild_id().unwrap().edit_nickname(Some(arg.as_str()));
+        }
+    }
```

Comment:
1. Why do all of these only work only apply to "allowed users"?
2. If you're constantly doing membership checks, why not put them in a `HashSet<T>`?